opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -70,9 +70,15 @@ def shutdown(self) -> None:
         """Called when a :class:`opentelemetry.sdk.trace.Tracer` is shutdown.
         """
 
-    def force_flush(self) -> None:
+    def force_flush(self, timeout_millis: int = 30000) -> bool:
```

Comment:
Any reason to have this timeout in millis rather than seconds? it looks like anywhere it's used gets turned into seconds anyways.

Reply:
I chose to use milliseconds because the schedule delay parameter for the batch span processor is in milliseconds. I don't like the idea of having a mix of seconds and milliseconds on the same class, what do you think?